samples/detect.v1p2beta1.js

```diff
@@ -24,31 +24,38 @@ function detectPdfText(bucketName, fileName) {
   // Creates a client
   const client = new vision.ImageAnnotatorClient();
 
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Bucket where the file resides, e.g. my-bucket';
+  // const fileName = 'Path to PDF file within bucket, e.g. path/to/document.pdf';
+
   const gcsSourceUri = `gs://${bucketName}/${fileName}`;
   const gcsDestinationUri = `gs://${bucketName}/${fileName}.json`;
 
-  console.log(gcsDestinationUri);
-
+  const inputConfig = {
+    // Supported mime_types are: 'application/pdf' and 'image/tiff'
+    mimeType: 'application/pdf',
+    gcsSource: {
+      uri: gcsSourceUri,
+    },
+  };
+  const outputConfig = {
+    gcsDestination: {
+      uri: gcsDestinationUri,
+    },
+  };
+  const features = [
```

Comment:
Nit: put all this on one line?

Reply:
+1 for that, but it will probably require prettier config change for this folder.

Reply:
So, yey or ney on this change?